packages/manager/src/features/LoadBalancers/LoadBalancerCreate/LoadBalancerCreate.tsx

```diff
@@ -26,35 +36,23 @@ const LoadBalancerCreate = () => {
         }}
         title="Create"
       />
-      <Stack spacing={3}>
-        <LoadBalancerLabel
-          labelFieldProps={{
-            disabled: false,
-            errorText: '',
-            label: 'Linode Label',
-            onChange: () => null,
-            value: '',
-          }}
-        />
-        <LoadBalancerRegions />
-        <LoadBalancerConfiguration />
-        {/* TODO: AGLB -
-         * Implement Review Load Balancer Action Behavior
-         * Implement Add Another Configuration Behavior
-         */}
-        <Box
-          columnGap={1}
-          display="flex"
-          flexWrap="wrap"
-          justifyContent="space-between"
-          rowGap={3}
-        >
-          <Button buttonType="outlined">Add Another Configuration</Button>
-          <Button buttonType="primary" sx={{ marginLeft: 'auto' }}>
-            Review Load Balancer
-          </Button>
-        </Box>
-      </Stack>
+      <Formik<FormValues>
```

Comment:
Can we go ahead and use `CreateLoadbalancerPayload`?

```suggestion
      <Formik<CreateLoadbalancerPayload>
```